lib/asn1/base/reporter.js

```diff
@@ -66,6 +66,10 @@ Reporter.prototype.error = function error(msg) {
     err = new ReporterError(state.path.map(function(elem) {
       return '[' + JSON.stringify(elem) + ']';
     }).join(''), msg.message || msg, msg.stack);
+
+    // IE only adds stack when thrown
+    if (!err.stack)
+      err.stack = err.message
```

Comment:
If it won't work on Edge (sorry, I can't test), I'd suggest replacing `throw this` with `throw new Error(this.message)` and adding `this.stack = e.stack` to `catch`.